### ISSUE TITLE
Lab 04, Exercise 1, Task 2. Provide powershell instruction into Note (contains Bash today) in lab.

### ISSUE THREAD

**Issue by jorhja** [MicrosoftLearning/AZ-104-MicrosoftAzureAdministrator#559]
#### Step: 05, Third Note

Description of issue
In the notes, regarding how to check for available vm-sizes in region, there a instructions how to do this with bash. But since the students are in a powershell session already, it would be better to update the note as below.
Repro steps:

1. OK, as is
2. Check the location in which the 'az104-04-rg1' resource group is deployed. You can run $Location = (Get-AzResourcegroup -name az104-04-rg1).Location in your CloudShell to put into the variable Location.
3. Run Get-AzVmSize -Location $Location | Where-Object {$_.Name -like "*Standard_D2s*"} in your CloudShell.
![PS note](https://user-images.githubusercontent.com/34010274/166904852-83f538c1-36ac-4dd0-b148-474dc9733ffa.PNG)
5. OK, as is
6. OK, as is

**Comment by craigbeeremct** [comment]
Its important to note that this does not look at what the sponsored trial can use; just what the Azure region can provide.

To find out what our subscription's quota is, use something like the following. I wrote these queries for the WS-050 course, which is why they include F-family vCPUs.

````pwsh
Get-AzVMUsage -Location $Location `
| Select-Object @{l="ResourceType";e={$PSItem.Name.LocalizedValue}}, CurrentValue, Limit `
| Where-Object { $PSItem.ResourceType -in "Standard DSv3 Family vCPUs", "Standard DSv5 Family vCPUs", "Standard FSv2 Family vCPUs", "Total Regional vCPUs" }


Get-AzComputeResourceSku -Location $Location `
| Where-Object { ($PSItem.Family -in "standardDSv3Family", "standardDSv5Family", "standardFSv2Family" ) -and ( $PSItem.ResourceType -eq "virtualMachines" ) } `
| Sort-Object Family, Name `
| ft Name, Family, @{l="Restriction";e={$PSItem.Restrictions.ReasonCode -join ", "}}
````

**Comment by craigbeeremct** [comment]
Here's a better example, focusing only on the D2s_v* machines.

````pwsh
Get-AzVMUsage -Location $Location `
| Select-Object @{l="ResourceType";e={$PSItem.Name.LocalizedValue}}, CurrentValue, Limit `
| Where-Object { $PSItem.ResourceType -in "Standard DSv3 Family vCPUs", "Standard DSv5 Family vCPUs", "Total Regional vCPUs" }

Get-AzComputeResourceSku -Location $Location `
| Where-Object { ($PSItem.Name -like "Standard_D2s*" ) -and ( $PSItem.ResourceType -eq "virtualMachines" ) } `
| Sort-Object Name `
| ft Name, @{l="Restriction";e={$PSItem.Restrictions.ReasonCode -join ", "}}
````

Students must pick a region where the **Limit** is greater than 0 and the **Restriction** is blank.

**Comment by MOC-Labs-Review** [comment]
MOC Labs Review has solved this issue with pull request #563.

**Comment by MOC-Labs-Review** [comment]
MOC Labs Review did not experience this issue
